src/module/iotjs_module_i2c.h

```diff
@@ -41,15 +41,16 @@ enum I2cError {
   kI2cErrOk = 0,
   kI2cErrOpen = -1,
   kI2cErrRead = -2,
-  kI2cErrWrite = -3,
+  kI2cErrReadBlock = -3,
+  kI2cErrWrite = -4,
 };
 
 struct I2cReqData {
   String device;
   char* buf_data;
-  int buf_len;
-  int8_t byte;
-  int8_t cmd;
+  uint8_t buf_len;
+  uint8_t byte;
+  uint8_t cmd;
   int32_t delay;
```

Comment:
For negative delay. For example, if you pass negative number to ``setTimeout``, it is changed to zero. Also, I think ``int32_t`` has enough range. (-2^31 ~ 2^31)